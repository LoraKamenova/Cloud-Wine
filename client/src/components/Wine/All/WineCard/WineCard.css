.wine-card {
    width: 320px;
    height: 450px ;
    background-color: white;
    margin: 10px;
    box-shadow: 0px 0px 11px -4px #000000;
}

.wine-card-image {
    width: 320px;
    height: 350px;
    margin: 0 auto;
    padding: 20px 10px 0 10px;
}

.wine-card-title-wrapper {
    text-align: center;
    font-family: 'Bad Script', cursive;
    font-size: 19px;
    margin: 15px 0;
    padding: 0 15px;
}
