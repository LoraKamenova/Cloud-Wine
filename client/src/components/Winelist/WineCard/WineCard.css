.wine-card {
    background-color: white;
    width: 320px;
    height: 400px ;
    margin: 10px;
    box-shadow: 0px 0px 11px -4px #000000;
}

.wine-card-image {
    margin: 0 auto;
    width: 320px;
    height: 320px;
    padding: 10px;
}

.wine-card-title-wrapper {
    font-size: 19px;
    font-family: 'Bad Script', cursive;
    text-align: center;
    margin: 15px 0;
}
