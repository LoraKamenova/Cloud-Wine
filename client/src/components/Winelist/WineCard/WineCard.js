import './WineCard.css'

const WineCard = (props) => {
    return (
        <article className="wine-card">
            <div className="wine-card-image-wrapper">
                <img className="wine-card-image"
                     src={props.imageUrl1} alt="wine item"/>
            </div>
            <div className="wine-card-title-wrapper">
                <h3 className="wine-card-title">{props.name}</h3>
            </div>
        </article>
    );
};

export default WineCard;
