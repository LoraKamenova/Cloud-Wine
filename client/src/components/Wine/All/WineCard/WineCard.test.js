import { BrowserRouter } from 'react-router-dom';
import { render } from '@testing-library/react';
import WineCard from './WineCard';

describe('WineCard Component', () => {
    it('Should display entry title', () => {
        render(
            <BrowserRouter>
                <WineCard name="Questionmark" link="/"/>
            </BrowserRouter>
        );

        expect(document.querySelector(".wine-card-title").textContent).toBe('Questionmark');
    });

    it("WineCard Component snapshot", () => {
        const { asFragment } = render(<BrowserRouter><WineCard link="/"/></BrowserRouter>);
        expect(asFragment()).toMatchSnapshot();
    });

});
