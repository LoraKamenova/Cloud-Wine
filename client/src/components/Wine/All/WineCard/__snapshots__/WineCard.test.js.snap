// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WineCard Component WineCard Component snapshot 1`] = `
<DocumentFragment>
  <article
    class="wine-card"
  >
    <div
      class="wine-card-image-wrapper"
    >
      <img
        alt="wine item"
        class="wine-card-image"
      />
    </div>
    <div
      class="wine-card-title-wrapper"
    >
      <h3
        class="wine-card-title"
      />
    </div>
  </article>
</DocumentFragment>
`;
